mne_bids/utils.py

```diff
@@ -163,9 +163,12 @@ def make_bids_folders(subject, session=None, kind=None, root=None,
     make_dir : bool
         Whether to actually create the folders specified. If False, only a
         path will be generated but no folders will be created.
-    overwrite : bool
-        If `make_dir` is True and one or all folders already exist,
-        this will overwrite them with empty folders.
+    overwrite : str, one of ('overwrite', 'error', 'clear')
+        How to handle overwriting previously generated data.
+        If overwrite = `overwrite` or `error` the previously created folder
+        will be left intact.
+        If overwrite = `clear` any existing folders at the session level or
+        lower will be removed, including any contained data.
```

Comment:
call the argument `write_mode` instead of `overwrite` ?

Reply:
yep, we should be consistent. Otherwise, new developers will have a hard time figuring out what is going on

Reply:
ok, sounds good to me.